winter/core/json/undefined.py

```diff
@@ -0,0 +1,4 @@
+class Undefined:
+    """A class to represent an absence of value."""
+    def __eq__(self, other):
```

Comment:
I think we should add `__hash__` too

Or rewrite it with enum (see other comments)

Reply:
Fixed it, thank you!